deps/wazuh_testing/wazuh_testing/modules/aws/event_monitor.py

```diff
@@ -62,6 +63,11 @@ def callback_detect_event_processed(line):
         return line
 
 
+def callback_detect_event_processed_or_skipped(pattern):
```

Comment:
Docstring is missing

Reply:
added in 11868d3ed5858c2f1c0f4f63bd254d45766c0565